x-pack/plugins/apm/public/components/app/service_map/Popover/stats_list.tsx

```diff
@@ -120,13 +132,20 @@ export function StatsList({ data, isLoading }: StatsListProps) {
         ),
         valueLabel: asPercent(memoryUsage?.value, 1, ''),
         timeseries: memoryUsage?.timeseries,
+        previousPeriodTimeseries: previousPeriod?.memoryUsage?.timeseries,
         color: 'euiColorVis8',
       },
     ],
-    [cpuUsage, failedTransactionsRate, memoryUsage, transactionStats]
+    [
+      cpuUsage,
+      failedTransactionsRate,
+      memoryUsage,
+      transactionStats,
+      previousPeriod,
+    ]
```

Comment:
It seems odd to pass in `currentPeriod` properties separately but not `previousPeriod`

```suggestion
    [
      currentPeriod,
      previousPeriod,
    ]
```